### ISSUE TITLE
Spoilers support (details)

### ISSUE THREAD

**Issue by LevYas** [whistyun/MdXaml#61]
Hello again!

Is it possible to make spoilers? I tried this syntax, but it didn't work:

```
<details>
  <summary>Spoiler warning</summary>
  
  Spoiler text. 
  
</details>
```

It works here:

<details>
  <summary>Spoiler warning</summary>
  
  Spoiler text. 
  
</details>

I tried with and without the MdXaml.Html plugin, and the result is the same.

**Comment by whistyun** [comment]
MdXaml.Html 1.19.0 has been released.
It enabled details tag.


![image](https://github.com/whistyun/MdXaml/assets/13712028/a919e367-2fb7-4135-be16-ad33df3f5931)  
![image](https://github.com/whistyun/MdXaml/assets/13712028/eb3e18e7-7098-4d88-b154-1e2076bace76)

**Comment by whistyun** [comment]
Thank you for trying the feature.
I tried your proposal, and realized that a link element on the summary is clickable.
So I will change most part of spoiler summary is clickable, but not some of it (e.g., link takes precedence over summary)

## sample

```md
<details>
<summary>Spoiler warning <a href="https://www.google.com">link</a>
</summary>
Spoiler text. Spoiler warning 
</details>
```
<details>
<summary>Spoiler warning <a href="https://www.google.com">link</a>
</summary>
Spoiler text. Spoiler warning 
</details>

## demonstration

![clickaction](https://github.com/whistyun/MdXaml/assets/13712028/17916335-6861-4a76-8a42-9c9b7255bbd6)

**Comment by whistyun** [comment]
MdXaml 1.19.1 has been released.
It enables spoiler header clicking to open and close details.

**Comment by whistyun** [comment]
> Is it possible to make the cursor a regular arrow?

I released MdXaml 1.19.2 to fix it.
